doc/source/quickstart.rst

```diff
@@ -407,21 +407,14 @@ and ``"key_3": 3``.
 
          void write_map() {
            tiledb::Context ctx;
-         
+
            // Open the map
            tiledb::Map map(ctx, map_name);
-         
```

Comment:
Also confused why these changes are here

Reply:
Probably out of habit I cleaned up the extra lined.

Reply:
Actually, now seeing this I am not sure. Let me check again in case I have not rebased properly.

Reply:
This is to sync up the quickstart docs with the changes made to to the corresponding code example. I can make a separate PR for these.